src/xenia/base/string_util.h

```diff
@@ -30,6 +34,40 @@
 namespace xe {
 namespace string_util {
 
+inline size_t copy_truncating(char* dest, std::string_view source,
```

Comment:
Should this not be `const std::string_view`? (and following instances here too)

Reply:
Didn't notice that, fixed!